cedar-policy-validator/src/human_schema/to_json_schema.rs

```diff
@@ -39,7 +39,7 @@ use super::{
     err::{schema_warnings, SchemaWarning, ToJsonSchemaError, ToJsonSchemaErrors},
 };
 
-/// Convert a custom schema AST into the JSON representation
+/// Convert a schema AST into the JSON representation
 /// This will let you subsequently decode that into the Validator AST for Schemas (`[ValidatorSchema]`)
```

Comment:
Because otherwise Markdown will run everything together into one sentence in the rendered docs
```suggestion
/// Convert a schema AST into the JSON representation.
/// This will let you subsequently decode that into the Validator AST for Schemas (`[ValidatorSchema]`).
```